ssts/segmentation/segmentation_watershed.py

```diff
@@ -86,7 +92,7 @@ def fit_transform(self, data, outliers=None):
 
     @staticmethod
     def normalize_data(data):
-        return (data - np.mean(data)) / np.std(data)
+        return np.abs(data) / np.max(np.abs(data))
```

Comment:
The normalized data ranges from 0 to 1. Thus a threshold of 0.5 is a "good" default parameter for the watershed algorithm.